opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableCreation.jsx

```diff
@@ -377,6 +552,55 @@ const StixCyberObservableCreation = ({
     );
   };
 
+  function BulkAdd(props) {
```

Comment:
Please move this component outside and independent. Right now it is using the state of the parent component directly and that's hard to read/debug

Reply:
Completed in latest commit. Please re-review to ensure the new component is sufficiently separate from the parent component. Thank you! 9199da1c3604c14e2ddbf1bcfd6f37b9e5c4a4e6

Reply:
I fixed some minor formatting and typing for this component in de7361299907a348c3c013bda4e9fd5c7128b56b